src/pages/DailyStatus.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState, useEffect, useCallback } from "react";
+import "../App.css";
+import "./home.css";
+import { DAILY_STATUS } from "../constants/constants";
+
+const DailyStatus = ({
+  state,
+  setState,
+  setPage,
+  page,
+}) => {
+  const [isSame, setIsSame] = useState(false);
+  const [isBetter, setIsBetter] = useState(false);
+  const [isWorse, setIsWorse] = useState(false);
+  const [statusError, setStatusError] = useState(false);
+
+  const handleOnStatusClick = useCallback(
+    (item) => () => {
+      const newState = { ...state, statusToday: item };
+      const isStatusChecked = Object.values(newState).some((s) => s);
+      setState(newState);
+      setStatusError(!isStatusChecked);
+    },
+    [state]
+  );
+
+  useEffect(() => {
```

Comment:
too complex logics, it can be simplified: 
this useEffect and isSame, isBetter, isWorse states are redundant as on each input we can check if they are checked in simple way, i.e. for DAILY_STATUS.same:
`checked={state["statusToday"] === DAILY_STATUS.same}`

Reply:
resolved